shared/src/business/useCases/practitioners/getPractitionersByNameInteractor.ts

```diff
@@ -1,21 +1,12 @@
-import { MAX_SEARCH_RESULTS } from '../../entities/EntityConstants';
 import {
   ROLE_PERMISSIONS,
   isAuthorized,
 } from '../../../authorization/authorizationClientService';
 import { UnauthorizedError } from '@web-api/errors/errors';
 
-/**
- * getPractitionersByNameInteractor
- *
- * @param {object} applicationContext the application context
- * @param {object} params the params object
- * @param {string} params.name the name to search by
- * @returns {*} the result
- */
 export const getPractitionersByNameInteractor = async (
   applicationContext: IApplicationContext,
-  { name }: { name: string },
+  { name, searchAfter }: { name: string; searchAfter: string },
 ) => {
```

Comment:
Can you explicitly type the return of this method please?

Reply:
will do!